battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java

```diff
@@ -35,10 +35,8 @@
     private Parent createContent() {
         BorderPane root = new BorderPane();
         root.setPrefSize(DEFAULT_ROOT_WIDTH, DEFAULT_ROOT_HEIGHT);
-        Button button = new Button();
-        button.setText(CommunicateProviderImpl.getCommunicate(Communicate.WRONG_IP));
```

Comment:
FYI, this is a literal translation of Polish "komunikat", but I don't think English has a noun `communicate` and quick check at Merriam-Webster confirms this. Consider checking few other dictionaries and proving me wrong or changing these class names to Message or Info or Missive or Communique (this one is pretty official).